practico_01/ejercicio-06.py

```diff
@@ -1,6 +1,8 @@
+
 # Implementar la función multiplicar() que devuelva el producto de todos los números de una lista.
 
 
 # [1,2,3,4] -> 24
 def multiplicar(lista):
-    pass
+   for i in range(len(lista)):
+       i*(i+1)
```

Comment:
Están multiplicando índices en lugar de valores, resolver con 

`for elem in list`

No usar `range `ni `len`